python/lsst/ts/cbp/component.py

```diff
@@ -195,24 +195,23 @@ async def move_azimuth(self, position: float):
         else:
             await self.send_command(f"new_az={position}")
 
-    async def get_altitude(self):
+    async def get_elevation(self):
```

Comment:
Thank you for switching to "elevation"

Please take another pass at the doc string. It uses "altitude" with no explanation. I suggest something along the lines of:
```
"""Read and record the mount elevation encoder, in degrees.

Note that the low-level controller calls this axis "altitude".
...
"""
```
Would it make any sense to also return the value (not unless you have a use case for that).

Reply:
I don't think so.